lib/galaxy/webapps/galaxy/api/configuration.py

```diff
@@ -85,7 +85,7 @@ def tool_lineages(self, trans):
 
     @expose_api
     @require_admin
-    def reload_toolbox(self, trans):
+    def reload_toolbox(self, trans, **kwds):
```

Comment:
I suppose this wasn't necessary in https://github.com/galaxyproject/galaxy/blob/dev/templates/admin/reload_tool.mako#L18 because javascript was handling the session?!